_includes/layouts/base.html

```diff
@@ -4,48 +4,49 @@
 {% endcomment %}
 <!DOCTYPE html>
 <html lang="en">
-
-{% if redirect_bases[tags[0]] and env.production %}
-  {% include "meta_redirect.html" %}
-{% elsif page.url == "/" and env.production %}
-<meta name="robots" content="noindex" />  
-<meta http-equiv="refresh" content="0;URL='https://18f.gsa.gov/guides'" />
-{% else %}
-  {% include "meta.html" %}
-  <body class="display-flex flex-column minh-viewport">
-    {% include "site-alert.html" %}
-    {% include "header.html" %}
-    {% if tags %}
-      {% assign primary_navigation = navigation[tags] %}
-    {% else %}
-      {% assign primary_navigation = site.primary_navigation %}
+  {% assign was_redirected = false %}
+  {% if env.production %}
+    {% assign is_index = page.url == "/" %}
+    {% if redirect_bases[tags[0]] or is_index %}
+      {% assign was_redirected = true %}
+      {% include "meta_redirect.html", is_index: is_index %}
     {% endif %}
+  {% endif %}
 
-    {% include "menu.html" secondary_navigation: site.secondary_navigation %}
+  {%  unless was_redirected %}
+    {% include "meta.html" %}
```

Comment:
I think this is where we need to adjust for the Federalist previews. The case where it's not `production`, which would be any pull request, we still want the `nofollow` meta. It seems like right now we don't address that case. We could either do that here, or add something like `{% unless env.production %}` within `meta.html` for the `nofollow` attribute